examples/react-native/src/Filters.js

```diff
@@ -9,14 +9,14 @@ import {
   TouchableHighlight,
   Keyboard,
 } from 'react-native';
-import { InstantSearch } from 'react-instantsearch/native';
+import { InstantSearch } from 'react-instantsearch-native';
 import {
   connectCurrentRefinements,
   connectMenu,
   connectRange,
   connectRefinementList,
   connectSearchBox,
-} from 'react-instantsearch/connectors';
+} from 'react-instantsearch-core';
```

Comment:
We should import from `react-instantsearch-native`. The `core` package is only used internally.